services/order.py

```diff
@@ -0,0 +1,39 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+def create_order(
+    tickets: list[dict],
+    username: str,
+    date: str = None
+) -> Order:
+    with transaction.atomic():
```

Comment:
In case all body of a method should be treated as an atomic transaction, it is better to use `transaction.atomic` as a decorator